mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -596,6 +605,10 @@ func resourceMongoDBAtlasAdvancedClusterRead(ctx context.Context, d *schema.Reso
 		return diag.FromErr(fmt.Errorf(errorClusterAdvancedSetting, "version_release_system", clusterName, err))
 	}
 
+	if err := d.Set("accept_data_risks_and_force_replica_set_reconfig", cluster.AcceptDataRisksAndForceReplicaSetReconfig); err != nil {
+		return diag.FromErr(fmt.Errorf(errorClusterAdvancedSetting, "accept_data_risks_and_force_replica_set_reconfig", clusterName, err))
+	}
+
```

Comment:
will this create an inconsistent plan? Since we were not returning it so far, I imagine the .tfstate doesn't have this file. After this new version, if customer does `terraform plan` will they encounter an unexpected behaviour? (I think it might be only if API returns some sort of default value...)

Reply:
I created the cluster with old code and applied with new code and no state change happened